docs/external-usage.md

```diff
@@ -0,0 +1,176 @@
+# Using Container Assist Tools in External MCP Servers
+
+This guide explains how to import and use Container Assist tools in your own MCP server implementation.
+
+## Quick Start
+
+```typescript
+import { 
+  configureTools, 
+  analyzeRepo, 
+  buildImage,
+  registerTool 
+} from '@thgamble/containerization-assist-mcp';
+import { Server } from '@modelcontextprotocol/sdk';
+
+// Create your MCP server
+const server = new Server({
+  name: 'my-server',
+  version: '1.0.0'
+});
+
+// IMPORTANT: Configure tools with your server for AI sampling
+configureTools({ server });
```

Comment:
does this use globals for routing the handlers? we could make a "ca-server" type that we instantiate and bind to a particular mcp to avoid the global state

Reply:
it's not even that so much, we just need the session to be shared